stores/suggestion_store.jsx

```diff
@@ -295,6 +295,7 @@ class SuggestionStore extends EventEmitter {
             this.setSuggestionsPending(id, false);
 
             if (this.isCompletePending(id)) {
+                this.setCompletePending(id, false);
```

Comment:
By clearing `completePending` flag, this allows the SuggestionList to appear. 

Previously, `completePending equaled true` which resulted in the SuggestionList not displaying for any providers {atMentions, channelMentions, commands, emoticons}